sites/public/pages/eligibility/income.tsx

```diff
@@ -49,13 +49,22 @@ const EligibilityIncome = () => {
   }
 
   function getFilterUrl() {
-    const params: ListingFilterState = {}
+    const state: ListingFilterState = {}
 
     if (eligibilityRequirements.age < SENIOR_AGE) {
-      params.seniorHousing = false
+      state.seniorHousing = false
     }
 
-    return `/listings/filtered?${encodeToFrontendFilterString(params)}`
+    // If the user has as a disability or they prefer not to reveal they're
+    // disability status, we don't need to filter the listings down further.
+    // We show all listings as well as the communities that require a disability
+    // status to apply.
+    if (eligibilityRequirements.disability === "false") {
+      state.specialNeedsHousing = false
+    }
+
+    return `/listings/filtered?${encodeToFrontendFilterString(state)}`
+    return `/listings?${encodeToFrontendFilterString(state)}`
```

Comment:
I think you just want the `filtered` version here?

Reply:
Done -- thanks!